clang/lib/Parse/ParseOpenACC.cpp

```diff
@@ -268,6 +270,71 @@ ExprResult Parser::ParseOpenACCRoutineName() {
   return getActions().CorrectDelayedTyposInExpr(Res);
 }
 
+void Parser::ParseOpenACCCacheVar() {
```

Comment:
Maybe it would be nice to do like C and C++, ie citing the standard + grammar production in a comment